modules/react/table/stories/stories_visualTesting.tsx

```diff
@@ -66,66 +67,45 @@ const exampleData = [
-            position="sticky"
-            left="0"
-            backgroundColor="soap100"
-            borderRight={`1px solid ${colors.soap400}`}
-          >
-            Header
-          </Table.Header>
-          <Table.Header borderRight={`1px solid ${colors.soap400}`} backgroundColor="soap100">
-            Header
-          </Table.Header>
-          <Table.Header borderRight={`1px solid ${colors.soap400}`} backgroundColor="soap100">
-            Header
-          </Table.Header>
-          <Table.Header borderRight={`1px solid ${colors.soap400}`} backgroundColor="soap100">
-            Header
-          </Table.Header>
-          <Table.Header borderRight={`1px solid ${colors.soap400}`} backgroundColor="soap100">
-            Header
-          </Table.Header>
-          <Table.Header borderRight={`1px solid ${colors.soap400}`} backgroundColor="soap100">
-            Header
-          </Table.Header>
-          <Table.Header borderRight={`1px solid ${colors.soap400}`} backgroundColor="soap100">
-            Header
-          </Table.Header>
-          <Table.Header borderRight={`1px solid ${colors.soap400}`} backgroundColor="soap100">
-            Header
-          </Table.Header>
-          <Table.Header borderRight={`1px solid ${colors.soap400}`} backgroundColor="soap100">
+          <Table.Header {...mergeStyles(tableHeaderStencil({variant: 'sticky'}))}>
             Header
           </Table.Header>
+          {[...Array(8).keys()].map(key => (
+            <Table.Header {...mergeStyles(tableHeaderStencil())}>Header</Table.Header>
+          ))}
         </Table.Row>
       </Table.Head>
       <Table.Body>
         {exampleData.map(item => (
           <>
             <Table.Row>
-              <Table.Header
-                position="sticky"
-                left="0"
-                backgroundColor="soap100"
-                borderRight={`1px solid ${colors.soap400}`}
-              >
+              <Table.Header {...mergeStyles(tableHeaderStencil({variant: 'sticky'}))}>
                 {item.header}
               </Table.Header>
-              <Table.Cell borderRight={`1px solid ${colors.soap400}`}>{item.cell}</Table.Cell>
-              <Table.Cell borderRight={`1px solid ${colors.soap400}`}>{item.cell}</Table.Cell>
-              <Table.Cell borderRight={`1px solid ${colors.soap400}`}>{item.cell}</Table.Cell>
-              <Table.Cell borderRight={`1px solid ${colors.soap400}`}>{item.cell}</Table.Cell>
-              <Table.Cell borderRight={`1px solid ${colors.soap400}`}>{item.cell}</Table.Cell>
-              <Table.Cell borderRight={`1px solid ${colors.soap400}`}>{item.cell}</Table.Cell>
-              <Table.Cell borderRight={`1px solid ${colors.soap400}`}>{item.cell}</Table.Cell>
-              <Table.Cell borderRight={`1px solid ${colors.soap400}`}>{item.cell}</Table.Cell>
+              {[...Array(8).keys()].map(key => (
+                <Table.Cell {...mergeStyles(tableHeaderStencil())}>{item.cell}</Table.Cell>
```

Comment:
Just for consistency, if you wanna implement the array the way Raisa suggested above

Reply:
Yeah I'm pushing that update now!